addons/binding/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/json/SenecHomeEnergy.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.senechome.internal.json;
+
+import java.io.Serializable;
+
+/**
+ * @author Steven.Schwarznau
```

Comment:
Please explain the class in a few words.

```suggestion
 * @author Steven Schwarznau - Initial Contribution
```

also applies to some other classes